planner/optimize.go

```diff
@@ -342,9 +371,18 @@ func getBindRecord(ctx sessionctx.Context, stmt ast.StmtNode) (*bindinfo.BindRec
 	if globalHandle == nil {
 		return nil, ""
 	}
-	bindRecord = globalHandle.GetBindRecord(hash, normalizedSQL, ctx.GetSessionVars().CurrentDB)
+	bindRecord = globalHandle.GetBindRecord(hash, normalizedSQL, "")
```

Comment:
Why we are using `""` instead of `CurrentDB` here?

Reply:
We use `Restore` instead of `Text`, which means that the `DB` information is in SQL, so we don't need to match the `DB`.